galaxy/api/internal/views/combined.py

```diff
@@ -0,0 +1,101 @@
+# (c) 2012-2019, Ansible by Red Hat
+#
+# This file is part of Ansible Galaxy
+#
+# Ansible Galaxy is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License as published by
+# the Apache Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Ansible Galaxy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# Apache License for more details.
+#
+# You should have received a copy of the Apache License
+from rest_framework import views
+from galaxy.main import models
+from rest_framework import response
+from galaxy.api.internal import serializers as internal_serializers
+from galaxy.api import serializers as v1_serializers
+
+
+class CombinedList(views.APIView):
```

Comment:
For an internal only API, if this view is only going to be collections and repositories, it might be more explicit to just call it something like ReposAndCollectionsList().  Assuming it's not going to gain support for another type of <whatever we call repositories and collections> that is. Not a big deal either way for an internal only api though.